tests/dualtor/test_standby_tor_upstream.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+import logging
+import ipaddress
+import json
+import re
+import time
+from tests.common.dualtor.dual_tor_mock import *
+from tests.common.helpers.assertions import pytest_assert as pt_assert
+from tests.common.dualtor.dual_tor_utils import rand_selected_interface, verify_upstream_traffic
+from tests.common.dualtor.mux_simulator_control import mux_server_url
+from tests.common.utilities import compare_crm_facts
+
+logger = logging.getLogger(__file__)
+
+pytestmark = [
+    pytest.mark.topology('t0'),
+    pytest.mark.usefixtures('apply_mock_dual_tor_tables', 'apply_mock_dual_tor_kernel_configs')
+]
+
+PAUSE_TIME = 10
+
+def get_l2_rx_drop(host, itfs):
+    """
+    Return L2 rx packet drop counter for given interface
+    """
+    res = {}
+    stdout = host.shell("portstat -j")['stdout']
+    match = re.search("Last cached time was.*\n", stdout)
+    if match:
+        stdout = re.sub("Last cached time was.*\n", "", stdout)
+    data = json.loads(stdout)
+    return int(data[itfs]['RX_DRP'])
+
+@pytest.fixture
+def clear_portstat(rand_selected_dut):
+    rand_selected_dut.shell("portstat -c")
+
+def test_standby_tor_upstream(rand_selected_dut, tbinfo, ptfadapter, clear_portstat, rand_selected_interface, apply_standby_state_to_orchagent):
```

Comment:
Please use use the other mocking fixtures such as  apply_mock_dual_tor_tables and apply_mock_dual_tor_kernel_configs for full mocking. (for all test cases)

Reply:
The mock fixtures are already used in line 19 at module level.

Reply:
Ah I missed that, thank you!